### DISCUSSION TITLE
cannot debug for lerna

### DISCUSSION THREAD

**Discussion by lingxuanHuang** [coder/code-server#4873] [Q&A]
I try to use  breakpoint debugging for lerna in code-server. But it never succeeded.  when I click run , the debugger tool  just flashed by, 

the launch,json like this.
![image](https://user-images.githubusercontent.com/17497665/154206818-7a822841-8fc5-4f45-9d43-c6930659de3d.png)

**Comment by lingxuanHuang** [comment]
it's the same problem with [#4464](https://github.com/coder/code-server/issues/4464) ,
I used 3.12.0,  downgrade to 3.11.1, it works.

**Reply by jsjoeio** [reply]
Did you try 4.0.2?

**Reply by lingxuanHuang** [reply]
yes, 3.11.1 has another problem, 4.0.2 can run perfectly!

**Reply by jsjoeio** [reply]
Hooray!! Thanks for testing.
